packages/core/src/build/static/ids.ts

```diff
@@ -1,45 +1,46 @@
 import crypto from "crypto";
 import type { Source } from "@/config/sources.js";
 import type { Schema } from "@/schema/types.js";
-import { isEnumColumn, isManyColumn, isOneColumn } from "@/schema/utils.js";
+import { isBaseColumn, isEnumColumn } from "@/schema/utils.js";
 import { dedupe } from "@/utils/dedupe.js";
 import type { TableAccess } from "./parseAst.js";
 
 export type FunctionIds = { [func: string]: string };
 export type TableIds = { [table: string]: string };
 
+type FunctionInputs = {
+  [key: string]: {
```

Comment:
Might be a matter of preference, but I think it's nice to name the key here like:
```ts
type FunctionInputs = {
  [functionId: string]: {
    astHash: string;
    sources: object[];
  }
};
```
Could even do something like this (not sure I like it):
```ts
type FunctionId = string

type FunctionInputs = {
  [id: FunctionId]: {
    astHash: string;
    sources: object[];
  }
};
```